spec/features/reviews/update_review_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+feature 'user creates review', %Q{
+  As an authenticated user
+  I want to update an a review for an airline
+  So that I can correct errors or provide new information
+} do
+  scenario 'user reviews updates sucessfully' do
+    user = FactoryGirl.create(:user)
+    FactoryGirl.create(:review)
+    visit new_user_session_path
+
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.